src/konserve/impl/defaults.cljc

```diff
@@ -34,7 +34,7 @@
   (-release [_this env]
     (if (:sync? env) nil (go-try- nil))))
 
-(defn key->store-key [key]
+(defn ^:dynamic key->store-key [key]
```

Comment:
Good question! I wanted to be able to use plain filenames instead of the general, but obscure UUID mapping, so I can open objects stored from Python under their filename and use [cbor2](https://github.com/agronholm/cbor2) to read them. This rebinding is not publicly documented and should only be used when you know what you are doing, i.e. you can create invalid filenames depending on your filesystem.

Reply:
This is just for me, I will remove it and use `alter-var-root` for now.